tests/Dockerfile.rpi4-BATS

```diff
@@ -1,43 +1,25 @@
 # Note RPi4 base WITH 64bit OS
 FROM balenalib/raspberrypi4-64-debian:bullseye-build
 
-RUN [ "cross-build-start" ]
+ENV DOCKER=1
 
-# Install systemd
-# Additionally, BATS tests on a minimal install will require additional packages
+# BATS tests on a minimal install will require additional packages
 # to run properly:
 # - lsb-release (influx, homegear)
 # - apt-transport-https (homegear, AdoptOpenJDK)
 # - unzip (java)
 # those packages would normally be included in our standard install
 RUN apt-get update -qq && \
-    apt-get install --yes -qq --no-install-recommends systemd systemd-sysv \
-        git wget python3 python3-pip apt-utils jq lsb-release unzip \
-        apt-transport-https gnupg acl && \
+    apt-get install --yes -qq --no-install-recommends git wget python3 \
+        python3-pip apt-utils jq lsb-release unzip apt-transport-https \
+        gnupg acl && \
     rm -rf /var/lib/apt/lists/*
 
-ENV APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=DontWarn
-ENV DOCKER=1
-
-RUN systemctl mask \
-    dev-hugepages.mount \
-    sys-fs-fuse-connections.mount \
-    sys-kernel-config.mount \
-    display-manager.service \
-    getty@.service \
-    systemd-logind.service \
-    systemd-remount-fs.service \
-    getty.target \
-    graphical.target \
-    kmod-static-nodes.service
-
-ENTRYPOINT ["./tests/entry.sh"]
-
 RUN git clone https://github.com/gdraheim/docker-systemctl-replacement && \
-    cp docker-systemctl-replacement/files/docker/systemctl3.py /bin/systemctl
+    cp docker-systemctl-replacement/files/docker/systemctl3.py /usr/bin/systemctl
```

Comment:
Err I'm not getting this.
You said you removed my 3rd party systemctl so why this ?

Reply:
Only in the non-BATS systems, see my comment on the Grafana for more info.